src/logicsig.ts

```diff
@@ -495,6 +495,29 @@ export function tealSign(
   return nacl.sign(toBeSigned, sk);
 }
 
+/**
+ * verifyTealSign verifies a signature as would the ed25519verify opcode
+ * @param sk - uint8array with public key to verify against
```

Comment:
I think this should be `pk` and at the bottom?

Reply:
You are right. I'll fix it